### DISCUSSION TITLE
Excel can't open exported files.

### DISCUSSION THREAD

**Discussion by stevex64** [PHPOffice/PhpSpreadsheet#2803] [Q&A]
I'm developing on the Symfony platform. I have tried several different ways to export both xlsx and ods files. They all open fine in LibreOffice Calc, but they won't open in Microsoft 365 Excel or the desktop app. Any advice or suggestions would be greatly appreciated.

**Comment by MarkBaker** [comment]
Not without seeing some examples: all the test files that I generate open cleanly in Excel; I'd need to know what features you might be using, how you were creating the files, what errors Excel might be showing when you tried to open them, etc

**Comment by lenilo1977** [comment]
try exporting older versions like .xls instead new .xlsx.
